PGD: added url for deb repo (#3224)
EnterpriseDB/docs: product_docs/docs/pgd/4/deployments/tpaexec/installing_tpaexec.mdx

The examples in this section assume you are using TPAexec v22.14 and installing Postgres 14.

1. Install the EnterpriseDB repository for TPAexec. 

>    - **To access the RPM repository:** Replace `token` with your company token and replace `pgVersion` with the version of Postgres you are installing.

Reviewer: ```suggestion
   - **To configure the yum repository (for RedHat derivatives):** Replace `token` with your company token.
```

     Syntax:
     ```

>      curl https://techsupport.enterprisedb.com/api/repository/<token>/products/tpa/release/<pgVersion>/rpm | bash

Reviewer: ```suggestion
     curl https://techsupport.enterprisedb.com/api/repository/<token>/products/tpa/release/14/rpm | bash
```

(It doesn't actually matter at all which version you mention here, so we can remove one unneeded replacement)

     ```

     Example:
     ```
     curl https://techsupport.enterprisedb.com/api/repository/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/products/tpa/release/14/rpm | bash
     ```

>     - **To access the Debian repository:** Replace `token` with your company token.

Reviewer: ```suggestion
    - **To access the apt repository (for Debian and Ubuntu):** Replace `token` with your company token.
```
